lib/inline_svg/railtie.rb

```diff
@@ -10,14 +10,12 @@ class Railtie < ::Rails::Railtie
 
     config.after_initialize do |app|
       InlineSvg.configure do |config|
-        # Configure the asset_finder:
-        # Only set this when a user-configured asset finder has not been
-        # configured already.
-        if config.asset_finder.nil?
-          # In default Rails apps, this will be a fully operational
-          # Sprockets::Environment instance
-          config.asset_finder = app.instance_variable_get(:@assets)
```

Comment:
Is there a reason not to call the `#assets` method here?

Reply:
This was to deal with older versions of Rails (probably back to 3.x) when these things did not all quack like ducks. May not be necessary with the supported versions of Rails.

Reply:
I replaced it with a method call.